msktkrb5.cpp

```diff
@@ -104,7 +104,7 @@ std::string get_salt(msktutil_flags *flags)
         if (flags->userPrincipalName.empty()) {
             salt = sform("%s%s",
                          flags->realm_name.c_str(),
-                         lower_accountname.c_str());
```

Comment:
With this consumer of `lower_accountname` gone, maybe move it from the start of the function to the only remaining user in the `else` branch below?